strato_query/__tests__/test_api_query.py

```diff
@@ -793,8 +795,8 @@ def test_job_runner_with_portfolio(self):
         # One line to run entire job pipeline
         df = SDJobRunner().load_df_from_job_pipeline(
             # Same job
-            portfolio_id='nk7M1E7A',
-            model_id='5ADyVKql',
+            portfolio_id='WMqA42Dw',
```

Comment:
Could you add a test with some sites please?

Reply:
I've added a test case, but I don't think it can pass until we update the API app.

Reply:
Good point. Probably should be wrapped up with https://github.com/StratoDem/SDDash/issues/2747 ?

Reply:
We'll need to update the strato-query package in that PR. Should I remove this test case so this can pass?